components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -233,8 +233,17 @@ public Response authorize(@Context HttpServletRequest request, @Context HttpServ
                 }
 
             } else if (resultFromLogin != null) { // Authentication response
-
+                long authTime;
+                if (FrameworkUtils.getSessionContextFromCache(FrameworkUtils.
```

Comment:
if statements are too long. break the statement and assign to necessary variables